sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CreateTest.java

```diff
@@ -316,6 +318,35 @@ public void testCreateGetName() {
     assertEquals("Create.TimestampedValues", Create.timestamped(Collections.EMPTY_LIST).getName());
   }
 
+
+  @Test
+  public void testCreateDefaultOutputCoderUsingInference() throws Exception {
+    Coder<Integer> coder = VarIntCoder.of();
+    PBegin pBegin = PBegin.in(p);
+    Create.Values<Integer> values = Create.of(1, 2, 3, 4, 5, 6, 7, 8);
+    Coder<Integer> defaultCoder = values.getDefaultOutputCoder(pBegin);
+    assertThat(defaultCoder, equalTo(coder));
+  }
+
+  @Test
+  public void testCreateDefaultOutputCoderUsingCoder() throws Exception {
+    Coder<Integer> coder = VarIntCoder.of();
+    PBegin pBegin = PBegin.in(p);
+    Create.Values<Integer> values = Create.of(1, 2, 3, 4, 5, 6, 7, 8).withCoder(coder);
+    Coder<Integer> defaultCoder = values.getDefaultOutputCoder(pBegin);
+    assertThat(defaultCoder, equalTo(coder));
+  }
+
+  @Test
+  public void testCreateDefaultOutputCoderUsingTypeDescriptor() throws Exception {
+    Coder<Integer> coder = VarIntCoder.of();
+    PBegin pBegin = PBegin.in(p);
+    Create.Values<Integer> values =
+        Create.of(1, 2, 3, 4, 5, 6, 7, 8).withType(new TypeDescriptor<Integer>() {});
```

Comment:
These would be best as something that can't use the default coder inference (e.g. `Record` and `Record2` which appear later in the test. You may have to write a `RecordCoder` that doesn't do anything.